src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -84,26 +97,69 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   } = useVenueEvents({
     venueIds: relatedVenueIds,
   });
-
   const isLoadingSchedule = isLoading || isEventsLoading;
 
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
+  const liveAndFutureEvents = useMemo(
+    () =>
+      relatedVenueEvents.map((event) =>
+        secondsToMilliseconds(
+          event.start_utc_seconds + minutesToSeconds(event.duration_minutes)
+        ) > startOfToday().getTime()
+          ? event.start_utc_seconds
+          : Number.MAX_SAFE_INTEGER
+      ),
+    [relatedVenueEvents]
+  );
+
+  const minDate = useMemo(
+    () =>
+      liveAndFutureEvents.length
+        ? Math.min(...liveAndFutureEvents)
+        : minRangeValue,
```

Comment:
can we write like this ?
```suggestion
      Math.min(...liveAndFutureEvents, minRangeValue)
```

Reply:
code block fixed in 85038ed